src/meson.build

```diff
@@ -54,3 +46,13 @@ zck_delta_size = executable(
     link_with: zcklib,
     install: true
 )
+if curl_dep.found()
+    zckdl = executable(
```

Comment:
The idea behind creating a disabler, is that you just create the executable() as normal.

If curl_dep is a disabler, then the disabler will be "viral" and anything that tries to link to it will instead become a disabler and be no-oped.

The exception is the found() method.

https://mesonbuild.com/Reference-manual_returned_disabler.html

Basically, you do not need the changes in this file.

Reply:
That was my understanding of how a disabler was supposed to work, but my first attempt was giving me compilation errors.  Fixing the code as you suggested in the previous comment made this work as expected.  Thanks!